README.md

```diff
@@ -168,7 +168,7 @@ Most dependencies can be installed using `apt-get install`:
 `sudo apt-get install g++ git automake libtool libgc-dev bison flex
 libfl-dev libgmp-dev libboost-dev libboost-iostreams-dev
 libboost-graph-dev pkg-config python python-scapy python-ipaddr python-ply
-tcpdump cmake`
+tcpdump cmake llvm`
```

Comment:
Would be nice to keep this in alphabetical order. I realize there was a previous offender, but it's not too hard to fix.